examples/pom.xml

```diff
@@ -24,7 +24,7 @@
     <maven.compiler.target>${java.version}</maven.compiler.target>
     <maven.deploy.skip>true</maven.deploy.skip>
     <spotbugs.fail>false</spotbugs.fail>
-    <opentelemetry.version>0.10.0</opentelemetry.version>
+    <opentelemetry.version>0.14.0</opentelemetry.version>
```

Comment:
I am keeping opentelemetry for example only. I am also upgrading it to the latest version.